server/lib/logging/route_logging.js

```diff
@@ -19,17 +19,31 @@ const disabled = function (req, res, next) {
   next();
 };
 
-const formats = {
-  'default_fxa': (tokens, req, res) => JSON.stringify({
+function defaultFxaFormat(tokens, req, res) {
+  'use strict';
+  let xff = (req.headers['x-forwarded-for'] || '').split(/\s*,\s*/);
+  xff.push(req.ip || req.connection.remoteAddress);
+  // Remove empty items from the list, in case of badly-formed header.
+  xff = xff.filter(x => x);
+  let clientAddressIndex = xff.length - (config.get('clientAddressDepth') || 1);
```

Comment:
Not sure about other servers, but let's fix this here. Can we move the `config.get` out of the `defaultFxaFormat` function. Assign it to a variable after `const config = require('../configuration');` above.
This way we don't have to do the `.get` lookup all the time.